src/display/editor/tools.js

```diff
@@ -441,14 +552,16 @@ class AnnotationEditorUIManager {
     );
   }
 
-  constructor(container, eventBus, annotationStorage) {
+  constructor(container, eventBus, pdfDocument, pageColors) {
     this.#container = container;
     this.#eventBus = eventBus;
     this.#eventBus._on("editingaction", this.#boundOnEditingAction);
     this.#eventBus._on("pagechanging", this.#boundOnPageChanging);
     this.#eventBus._on("scalechanging", this.#boundOnScaleChanging);
     this.#eventBus._on("rotationchanging", this.#boundOnRotationChanging);
-    this.#annotationStorage = annotationStorage;
+    this.#annotationStorage = pdfDocument?.annotationStorage;
```

Comment:
Given the line just below, we can probably access this directly?
```suggestion
    this.#annotationStorage = pdfDocument.annotationStorage;
```